packages/fhir-converter/src/lib/handlebars-converter/handlebars-helpers.js

```diff
@@ -149,6 +149,22 @@ var alreadyValidDateTime = function (dateTimeString) {
 
 // handling the date format here
 var getDate = function (dateStringRaw) {
+  if (dateStringRaw instanceof Date) {
+    console.log(
```

Comment:
leftover console logs throughout

Reply:
Intentional, so we can see what the values are and address them on the call to this function, otherwise we'll be losing data that we might be able to transform.